### ISSUE TITLE
XFCE PolicyKit Agent Error

### ISSUE THREAD

**Issue by HugoPlacer** [archcraft-os/archcraft#159]
When I login in Openbox appears this error, how I can solve it? What's happening?

Thanks :)

![Screenshot_2021-06-25-44_646x125](https://user-images.githubusercontent.com/69766683/123415466-95907000-d5b5-11eb-9a50-331bff6836b6.png)

**Comment by mySaYo** [comment]
I also met.

**Comment by adi1090x** [comment]
Fixed!

**Comment by riquezjp** [comment]
@adi1090x In case this is useful ~

I began having this PolicyKit error window in Archcraft 2023.01.20 ISO yesterday.
I wondered if software I had installed may have triggered it, so I removed 2 things I wasn't using.
**Khal & gThumb**
After removing them, the policykit error went away.

`sudo pacman -Rs khal gthumb`

Additionally, I was also experiencing 2 extra polybar icons on the far-right not part of the polybar config. A second network menu & Bluetooth menu. After removing the above packages these icons also disappeared.
